package.json

```diff
@@ -47,6 +47,7 @@
     "debug": "^4.3.4",
     "drupal-jsonapi-params": "^2.0.0",
     "jest-axe": "^6.0.0",
+    "jsona": "^1.11.0",
```

Comment:
@tjheffner Am I recalling that this was something you were wrestling with previously? I got some weirdness around this, but I thought this was already in `package.json`. Do you recall?

Reply:
Actually, I'm remembering now that my branch was based on yours, so that's probably why it's showing up here.

Reply:
. It was a sub-dependency in our installed modules already, but because we have to deserialize paginated responses manually in `data/queries/storyListing` (`TJsonaModel`) I needed to explicitly add it as a dependency to satisfy eslint